Components/ListSearchResults/Partials/Results/_style.scss

```diff
@@ -1,14 +1,20 @@
 .searchList {
+  background-color: $color-background;
+  border-radius: 4px;
+  box-shadow: 0 1px 26px 0 rgba(0, 0, 0, 0.06);
   list-style: none;
+  margin-top: 1.25rem;
```

Comment:
I think this should be 20px instead, to make it consistent with the padding of the pagination used later.